gslab_scons/builders/build_tables.py

```diff
@@ -52,8 +52,8 @@ def do_call(self):
         '''
         '''
         output = tablefill(input    = self.input_string, 
-                           template = self.source_file, 
-                           output   = self.target_file)
+                           template = os.path.normpath(self.source_file), 
+                           output   = os.path.normpath(self.target_file))
```

Comment:
Are you sure we need to norm these paths?

Reply:
for consistency + better safe than sorry

Reply:
I checked the code of [tablefill.py](https://github.com/gslab-econ/gslab_python/blob/master/gslab_fill/tablefill.py) and the only time the paths are used is to pass to opening the file (no manual string manipulation), so there should be no worry about platforms for this one.